src/views/Team/Outlet.vue

```diff
@@ -2,97 +2,189 @@
+                  :icon="editIcon"
+                  data-cy-role="editModal"
+                  v-if="canEdit"
+                  color="light"
+                  @click="intendEditTitle"
+                />
               </h1>
+
               <inline-link-list :items="socialLinks" showIcon>
-                <li v-if="canEdit">
-                  <ion-button @click="intendEditSocialLink" color="light" size="small" fill="clear">
-                    <ion-icon size="small" :icon="editIcon" />
-                  </ion-button>
-                </li>
+                <div v-if="canEdit" style="display: inline">
+                  <ion-icon
+                    size="small"
+                    :icon="editIcon"
+                    @click="intendEditSocialLink"
+                    color="light"
+                  />
+                </div>
               </inline-link-list>
+
+              <div class="extra-actions" v-if="canEdit">
+                <ion-chip
+                  title="remove background"
+                  v-if="team.background"
+                  @click="removeBackground"
+                  outline
+                >
+                  <ion-icon :icon="imageIcon" />
+                  <ion-icon :icon="trashIcon" />
+                </ion-chip>
+
+                <ion-chip
+                  v-else
+                  @click="selectBackground"
+                  title="upload background"
+                >
+                  <ion-icon :icon="imageIcon" />
+                  <ion-icon :icon="uploadIcon" />
+                </ion-chip>
+                <inline-link-list showTitle :items="footerLinks">
+                  <li v-if="canEdit">
+                    <ion-button
+                      @click="intendEditFooterLinks"
+                      size="small"
+                      fill="clear"
+                    >
+                      <ion-icon size="small" :icon="editIcon" />
+                    </ion-button>
+                  </li>
+                </inline-link-list>
+              </div>
+            </ion-col>
+          </div>
+
+          <ion-toolbar>
+            <!-- Scrollable Segment -->
+            <ion-segment
+              scrollable
+              value="about"
+              mode="md"
+              @ionChange="segmentChanged($event)"
+            >
+              <ion-segment-button value="qrcode">
+                <ion-icon :icon="qrCodeIcon" />
+              </ion-segment-button>
+              <ion-segment-button value="about">
+                <ion-label>About</ion-label>
+              </ion-segment-button>
+              <ion-segment-button value="news">
+                <ion-label>News</ion-label>
+              </ion-segment-button>
+              <ion-segment-button value="feed">
+                <ion-label>Feed</ion-label>
+              </ion-segment-button>
+            </ion-segment>
+          </ion-toolbar>
+
+          <div style="display: block; overflow-y: auto; height: 60vh">
```

Comment:
rather than inlining styles, let's use the `scoped`-styles per each component at the end section. So here, give this an internal class-name and add the style blow.